main.go

```diff
@@ -181,6 +188,29 @@ func initSelfNodeRemediationManager(mgr manager.Manager) {
 		setupLog.Error(err, "failed to add template creator to the manager")
 		os.Exit(1)
 	}
+
+	myNodeName := os.Getenv(nodeNameEnvVar)
+	if myNodeName == "" {
+		setupLog.Error(errors.New("failed to get own node name"), "node name was empty",
```

Comment:
Is it ok to continue if it cannot find the node's name?
IIUC, for Agents it quite necessary

Reply:
nevermind, just noticed that we're initializing the manager here :)

Reply:
I agree, but I think the location is wrong - this is where the manager is initilized. 
The fix should be where the agent is initialized: [here](https://github.com/medik8s/self-node-remediation/pull/141/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L205-R239)